.circleci/config.yml

```diff
@@ -202,10 +202,17 @@ commands:
       module_target:
         description: module target
         type: string
+      inject_token:
+        description: whether to inject an access token file
+        type: boolean
+        default: false
     steps:
       - run:
           name: Assemble Instrumentation Test APK for << parameters.module_target >>
           command: |
+            if << parameters.inject_token >>; then
+                echo "${MAPBOX_DEVELOPER_CONFIG}" > /root/code/<< parameters.module_target >>/src/androidTest/res/values/mapbox_access_token.xml
```

Comment:
Should we put this token in the file? Can we use it from env var only?

Reply:
We don't want to keep tokens in our public repo. I've added an ignored mapbox_access_token file for local use and reused the approach of injecting the token on CI from other tasks.